charts/metrics-server/values.yaml

```diff
@@ -103,6 +103,19 @@ service:
   #  kubernetes.io/cluster-service: "true"
   #  kubernetes.io/name: "Metrics-server"
 
+addonResizer:
+  enabled: false
+  nanny:
+    cpu: 20m
+    extraCpu: 1m
+    extraMemory: 2Mi
+    memory: 15Mi
+    minClusterSize: 10
+    pollPeriod: 300000
+    threshold: 5
+  image: k8s.gcr.io/addon-resizer:1.8.11
```

Comment:
It would be useful to have the tag separate. It likely also makes sense to default to the [latest version](https://github.com/kubernetes/autoscaler/releases/tag/addon-resizer-1.8.14) #846 changed that in the manifests - the registry also seems to have changed.

Reply:
That sounds good to me too.